src/kernel/arch/x86/arch.zig

```diff
@@ -45,7 +45,7 @@ const MemProfile = @import("../../mem.zig").MemProfile;
 ///
 /// Initialise the architecture
 ///
-pub fn init(mem_profile: *const MemProfile, allocator: *std.mem.Allocator) void {
+pub fn init(mem_profile: *const MemProfile, allocator: *std.mem.Allocator, rt_test: bool) void {
```

Comment:
So add a extern to the rt_test so don't need an additional parameter

Reply:
Also, if for ever reason need to add more build options, then these parameters could get very long

Reply:
How about passing the config struct instead of passing each individual option? I think that would be better than the current implementation at least.

Reply:
yeh, that would be fine